diagnostic.md

```diff
@@ -70,15 +62,11 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+Absolute because the use of the command doesnt start from the root.
```

Comment:
Relative path, planning is referenced with respect to the current working directory, project.